google/cloud/bigtable/tests/admin_integration_test.cc

```diff
@@ -161,11 +166,11 @@ TEST_F(AdminIntegrationTest, DropAllRowsTest) {
   };
 
   // Create records
-  CreateCells(*table, created_cells);
+  CreateCells(table, created_cells);
   // Delete all the records from a table
   EXPECT_STATUS_OK(table_admin_->DropAllRows(table_id));
-  auto actual_cells = ReadRows(*table, bigtable::Filter::PassAllFilter());
-  EXPECT_STATUS_OK(DeleteTable(table_id));
+  auto actual_cells = ReadRows(table, bigtable::Filter::PassAllFilter());
+  EXPECT_STATUS_OK(table_admin_->DeleteTable(table_id));
```

Comment:
I think this test could use `GetTable()`.

Reply:
Done.